server/models/users.py

```diff
@@ -17,26 +17,19 @@ class UserModel(db.Model):
     prospects = db.relationship('ProspectModel', backref='user', lazy=True)
     created = db.Column(db.DateTime, nullable=False, default=datetime.datetime.utcnow)
 
-    def __init__(self, email, password, credentials, firstName, lastName):
+    def __init__(self, email, password, firstName, lastName):
         self.email = email
         self.password = flask_bcrypt.generate_password_hash(
             password, create_app().config.get('BCRYPT_LOG_ROUNDS')
         ).decode()
         self.firstName = firstName
         self.lastName = lastName
-        self.credentials = credentials
 
     def save_to_db(self):
         # Session can get into a weird state where nothing else works?
         # See https://stackoverflow.com/questions/8870217/why-do-i-get-sqlalchemy-nested-rollback-error
         db.session.add(self)
-        try:
-            db.session.commit()
-        except:
-            db.session.rollback()
-            raise
-        finally:
-            db.session.close()
+        db.session.commit()
```

Comment:
Are you sure that the try/except was the problem? Did you re-test now that you removed those lines of code?

Reply:
Yes I have tested it after i removed the lines. I think it mostly has to do with close. Just because in the registration we make an auth token and we use the UserModel to do that and if we close the session i THINK it turns it off so we can't do it?